dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -3403,6 +4564,64 @@
       },
       "x-ms-client-name": "SingleLabelClassificationEvaluationSummary"
     },
+    "TextAnalysisAuthoringSpanSentimentEvaluationSummary": {
+      "description": "Represents the evaluation summary for a custom sentiment project.",
+      "required": [
+        "confusionMatrix",
+        "sentiments",
+        "microF1",
+        "microPrecision",
+        "microRecall",
+        "macroF1",
+        "macroPrecision",
+        "macroRecall"
+      ],
+      "type": "object",
+      "properties": {
+        "confusionMatrix": {
+          "description": "Represents the confusion matrix between two sentiments (the two sentiments can be the same). The matrix is between the sentiment that was labelled and the sentiment that was predicted.",
+          "$ref": "#/definitions/TextAnalysisAuthoringConfusionMatrix"
+        },
+        "sentiments": {
+          "description": "Represents the sentiment evaluation",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TextAnalysisAuthoringSentimentEvaluationSummary"
+          }
+        },
+        "microF1": {
+          "format": "float",
+          "description": "Represents the micro F1",
+          "type": "number"
+        },
+        "microPrecision": {
+          "format": "float",
+          "description": "Represents the micro precision",
```

Comment:
I recommend documenting the expected range like from 0..1

Reply:
Same for all floats. What the numbers represent is important for understanding the API.